dotcom-rendering/src/components/FeatureCard.tsx

```diff
@@ -381,7 +375,12 @@ export const FeatureCard = ({
 											roundedCorners={false}
 											aspectRatio={aspectRatio}
 										/>
-										{showPlayIcon &&
+										{/**
+										 * If the card isn't playable, we need to show a play icon.
+										 * Otherwise, this is handled by the YoutubeAtom.
```

Comment:
This can be deleted, we no longer show play icons on cards that aren't playable.